src/main/java/org/snomed/aag/rest/ServiceIntegrationController.java

```diff
@@ -63,12 +64,30 @@ public ServiceIntegrationController(CommitInformationValidator commitInformation
 
 			return ResponseEntity.status(HttpStatus.OK).build();
 		} else {
-			boolean pacComplete = projectAcceptanceCriteriaService.incrementIfComplete(commitInformation);
+			String sourceBranchPath = commitInformation.getSourceBranchPath();
+			ProjectAcceptanceCriteria projectAcceptanceCriteria = projectAcceptanceCriteriaService.findEffectiveCriteriaWithMandatoryItems(sourceBranchPath);
+			if (projectAcceptanceCriteria == null) {
+				String message = String.format("No Project Acceptance Criteria found for branch %s. Returning %s.", sourceBranchPath, HttpStatus.NO_CONTENT);
+				logger.info(message);
+				return ResponseEntity
+						.status(HttpStatus.NO_CONTENT)
+						.body(message);
+			}
+
+			if (!projectAcceptanceCriteria.hasCriteria()) {
```

Comment:
Is this possible? If there are any mandatory items (like classification) this I assume hasCriteria() will return true? .. it probably should, I expect many projects will use the defaults.

Reply:
I agree it is overkill. 

I've done some testing, and if there are no criteria items (even if no mandatory), the request will eventually fail because the criteria is empty. But this is a super edge case of where there's no data in the store. 

I'll remove the check from the controller.

Reply:
Thanks for helping cull excessive code Kai. 

I've made that change.